doc/nrf/known_issues.rst

```diff
@@ -327,9 +328,9 @@ nRF5
 nRF5340
 =======
 
-.. rst-class:: v2-0-2 v2-0-1 v2-0-0
+.. rst-class:: v2-1-0 v2-0-2 v2-0-1 v2-0-0
```

Comment:
Fixed for 2.1.0 - please remove the new tag.

Reply:
Updated.